app/src/main/res/layout/fragment_map.xml

```diff
@@ -13,12 +13,14 @@
 
     <Button
         android:id="@+id/logoutBtn"
+        style="@style/PrimaryButton"
```

Comment:
You can avoid setting the primary button style of your app by adding this field to your theme 
`<item name="materialButtonStyle">@style/PrimaryButton</item>`